### ISSUE TITLE
Bug: Very few position reports from San Diego MTS (278) making it to gtfs_rt.vehicle_positions

### ISSUE THREAD

**Issue by edasmalchi** [cal-itp/data-infra#1086]
**Describe the bug**
There is very little MTS (itp_id 278) vehicle_positions data in gtfs_rt.vehicle_positions, covering only about an hour on two separate days over the last week.

**To Reproduce**
On BigQuery:
```
        SELECT calitp_itp_id, calitp_url_number,
        -- header.timestamp AS header_timestamp, vehicle.timestamp AS vehicle_timestamp,
        -- vehicle.vehicle.label AS entity_id, vehicle.vehicle.id AS vehicle_id,
        -- vehicle.trip.tripId AS trip_id, vehicle.position.longitude AS vehicle_longitude,
        -- vehicle.position.latitude AS vehicle_latitude,
        MIN(EXTRACT (TIME FROM TIMESTAMP_SECONDS(vehicle.timestamp) AT TIME ZONE "America/Los_Angeles")) AS pst_time_min,
        MAX(EXTRACT (TIME FROM TIMESTAMP_SECONDS(vehicle.timestamp) AT TIME ZONE "America/Los_Angeles")) AS pst_time_max,
        EXTRACT(DAY FROM TIMESTAMP_SECONDS(vehicle.timestamp) AT TIME ZONE "America/Los_Angeles") AS pst_day
        FROM `cal-itp-data-infra.gtfs_rt.vehicle_positions`
        WHERE calitp_itp_id = 278
        GROUP BY pst_day, calitp_itp_id, calitp_url_number
        LIMIT 1000
```

**Expected behavior**
Fully populated vehicle_positions data for this operator.

**Additional context**
(query results)
<img width="475" alt="Screen Shot 2022-02-14 at 5 07 35 PM" src="https://user-images.githubusercontent.com/54728455/153973223-3fa6967c-744b-4745-8dfe-39e359bc1082.png">

**Comment by holly-g** [comment]
This bug pertains to Eric's D11 presentation on March 1.  Let's call this a `P1` (high priority, requires timely attention) since it's blocking his work; we'll bump this to the team's plate for the week and discuss tomorrow what will have to be depriortized

**Comment by lauriemerrell** [comment]
Initial diagnosis: This is *not* an issue with data that we have downloaded not getting into BigQuery. Looking at the raw-ish JSON files for San Diego, they seem to only have data in these time windows. And we have not successfully gotten data for them every day -- most recent data is 2/11. I am wondering if these are issues with their feed, I'm trying to experiment with the RT validator to ping their URLs but am having difficulties... More to come.

**Comment by lauriemerrell** [comment]
Immediate issue here was resolved by #1088.

Longer term work to resolve the conflict will be tracked in #1091

**Comment by lauriemerrell** [comment]
@atvaccaro do you want to close this once you've verified that we successfully pull data throughout the day today?
